core/src/themes/md/high-contrast-dark.tokens.ts

```diff
@@ -0,0 +1,58 @@
+import { highContrastDarkTheme as baseHighContrastDarkTheme } from '../base/high-contrast-dark.tokens';
+import type { HighContrastDarkTheme } from '../themes.interfaces';
+
+export const highContrastDarkTheme: HighContrastDarkTheme = {
+  ...baseHighContrastDarkTheme,
+
+  backgroundColor: '#121212',
+  textColor: '#000000',
+
+  // TODO(FW-6864): Remove once IonToolbar themes are added
+  toolbar: {
+    background: '#1f1f1f',
+  },
```

Comment:
Tests fail when `--ion-toolbar-background` is not set in certain cases like `md/dark`. Since the goal of this PR is to enable the palettes, I didn't think create this variable under the components since as of now it would have rendered it as `--ion-components-ion-toolbar-background`. I would have had to either update the generate function to not add the `--ion-components` suffix or update all the `--ion-toolbar-background` to the new variable if I wanted to do it properly. That would have been outside of the scope so this approach was taken along with adding a TODO to come back.